.github/workflows/master.yml

```diff
@@ -0,0 +1,33 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
```

Comment:
Can this be changed to Debian Stretch? It's what we currently run on.

Reply:
GitHub actions do not support Debian Stretch.

I can change it to Ubuntu 16.04, which is based off of Stretch according to https://askubuntu.com/questions/445487/what-debian-version-are-the-different-ubuntu-versions-based-on

Reply:
Yeah, that sounds good. Took a cursory look and most package versions are the same as on debian, which is good.